jena-core/src/test/java/org/apache/jena/reasoner/test/TestRDFSReasoners.java

```diff
@@ -95,6 +95,7 @@ public static TestSuite suite() {
     /**
      * Build a single named query test
      */
+    @SuppressWarnings("unused")
     private static void constructSingleQuerytests(TestSuite suite, String manifest, String test, ReasonerFactory rf, Resource config) throws IOException {
```

Comment:
No other change in this file.

And it introduces a warning when eclipse is set to not to have "unused" warnings. (Seems to be a "no win" situation in Eclipse.)  As Jena has a lot of warnings if "unused" warnigns are turned on, for now, can be just drop this annotation.